feature-utils/poly-look/test/react-components/legend.test.js

```diff
@@ -0,0 +1,43 @@
+import {
+  LegendEntry,
+  Legend,
+  IncorrectLegendEntry,
+} from "../../src/react-components/legend.js";
+
+describe("LegendEntries work", () => {
+  it("Creates a legendEntry", () => {
+    const noColor = "#000000";
+    const noColorStr = "No color";
+    const legendEntry = new LegendEntry(noColorStr, noColor);
+    expect(legendEntry).toBeTruthy;
+    expect(legendEntry).toBeInstanceOf(LegendEntry);
```

Comment:
So, is there _any_ way in JavaScript how `new LegendEntry()` could _not_ return an object that is both truth-y, and an instance of `LegendEntry`? While I'm only 99% sure, I'd be willing to bet some money on that _not_ being possible :D I like to think: If I don't know how to make an assert _fail_, I will not include it. So how to make this one fail?

Reply:
Yes, by the object constructor raising an exception for some reason. Any way, they are minimally viable products. They don't test extensively the behavior, or the DOM that's generated. You're right, they're not terribly useful. But we can improve them in time, I guess.

Reply:
Of course you're right. But once again, the scope of this PR was basically go from no tests to some tests. There's not a lot of logic to be covered either. But it's true that they can easily be erased, and substitute them for whatever logic there actually is.